packages/cookie-universal-nuxt/types/index.d.ts

```diff
@@ -29,8 +29,14 @@ export interface NuxtCookies {
   removeAll: () => void;
 }
 
-declare module '@nuxt/vue-app/types' {
-  interface NuxtAppOptions {
-    $cookies: NuxtCookies
+declare module "@nuxt/vue-app" {
```

Comment:
Can we change the whole block to the following? I've tested it and it seems to be working all right:
```ts
declare module "@nuxt/vue-app" {
  interface Context {
    $cookies: NuxtCookies
  }
  interface NuxtAppOptions {
    $cookies: NuxtCookies;
  }
}
// Nuxt 2.9+
declare module "@nuxt/types" {
  interface Context {
    $cookies: NuxtCookies
  }
  interface NuxtAppOptions {
    $cookies: NuxtCookies;
  }
}
```

Reply:
@microcipcip  Yeah no problem! I forgot it could be in Context too :)

Reply:
I'm pretty new to contributing, do I have to do the changes or accept it in some place?